healthScore/requirements.txt

```diff
@@ -1,21 +1,35 @@
 asgiref==3.7.2
 awsebcli==3.20.10
+black==24.2.0
 blessed==1.20.0
 botocore==1.31.85
 cement==2.8.2
 certifi==2024.2.2
 charset-normalizer==3.3.2
+click==8.1.7
 colorama==0.4.3
+coverage==6.5.0
+coveralls==3.3.1
 distlib==0.3.8
 Django==4.2
+docopt==0.6.2
 filelock==3.13.1
+flake8==7.0.0
 idna==3.6
 jmespath==1.0.1
+mccabe==0.7.0
+mypy-extensions==1.0.0
+mysql-connector==2.2.9
```

Comment:
Please remove these as they might cause deployment issues

Reply:
Right, updating the PR